frontend/src/contracts.ts

```diff
@@ -368,6 +369,10 @@ export async function setUpContracts(web3: Web3): Promise<Contracts> {
   const treasuryContractAddr = getConfigValue('VUE_APP_TREASURY_CONTRACT_ADDRESS') || (treasuryNetworks as Networks)[networkId]!.address;
   const Treasury = new web3.eth.Contract(treasuryAbi as Abi, treasuryContractAddr);
 
+  const multicallAddr = getConfigValue('VUE_APP_MULTICALL_CONTRACT_ADDRESS');
```

Comment:
Do we want to keep line 109
`if(process.env.NODE_ENV === 'development') return '';`
This was originally commented out, and it isn't in CB-Marketplace, which is the only reason this would work in a dev environment.

Reply:
We need to leave it, as it's in place for taking the address from generated abi files (as the addresses change every time you setup a dev env). I don't think we have multicall in our migrations, so it won't load it correctly. We somehow need to address this issue,